BlueprintUICommonControls/Tests/Sources/BoxTests.swift

```diff
@@ -18,13 +18,31 @@ class BoxTests: XCTestCase {
             identifier: "clear")
     }
 
-    func test_cornerRadius() {
```

Comment:
Could you delete the old image please?

Reply:
I think you mean this file, which has been deleted, though it's a bit tricky to tell from Github.

<img width="1088" alt="Screen Shot 2020-09-13 at 11 55 12 PM" src="https://user-images.githubusercontent.com/1542313/93042283-a3739680-f61c-11ea-81ea-2c1890eb01c6.png">

Please let me know if you're referring to another one.

Reply:
Oh oops, yeah sorry. GitHub on my phone really didn't want me to see it. Thanks!